assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+
+## RUP
+
+| Cmmi Process Area | Covered | Rationale  |
+| ------------------| ------- | ---------  |
+| **Level 2**           |
+| CM                | Fully | RUP's configuration and change management fully covers all of this process area's goals            |
+| MA                | Partially | RUP's project management work flow involves an activity for creating a measurement plan although RUP does not exactly specify how this should be communicated to all relevant parties            |
+| PMC               | Fully | The work flow pertaining to project management covers all aspects of monitoring a project against the plan and taking necessary corrective actions              |
+| PP                | Partially | The project management and environment work flows provide adequate support to the project planning with the exception of determining non-software related project attributes such as labor requirements.                 |
+| PPQA              | Fully | The review record in each of RUP's work flow addresses the issue of quality assurance and the creation of the problem resolution plan insures that all quality issues are dealt with.            |
+| REQM              | Fully | The requirements work flow integrates change management along with the capturing and analyzing of requirements.            |
+| SAM               | Not at all | RUP has no elements which dictate interactions with external suppliers it use purely centered around development            |
+| **Level 3**           |
+| DAR               | Not at all | This process area seems to not be covered in RUP in any work flow            |
+| IPM               | Fully | The project management and environment work flows encourages the creation of integrated product plans but are lacking when it comes to the management of dependencies.            |
+| OPD               | Not at all | Due to it being a project methodology and RUP does not prescribe any method for dealing with organizational process assets.             |
+| OPF               | Partially | RUP is a roject methodology so organizations would need to supplement it with other processes for this area. Although, RUP's environment work flow does have an activity which handles the assessment of the development-organization.            |
+| OT                | Not at all | Identifying training needs of an organization and assessing its effectiveness is not covered in RUP.          |
+| PI                | Fully | RUP addresses product integration through its implementation, test and deployments workflows.            |
+| RD                | Fully | Mainly through its Requirements workflow, but also through others, RUP provides good support for understanding stakeholder needs, produce and refine requirements and manage change requests. |
```

Comment:
Requirements shouldn't be capitalized. Can you name two or three of the others?  I'm not quite sure what this sentence is supposed to say, but the part after stackeholder needs doesn't flow very well.